README.md

```diff
@@ -434,11 +434,11 @@ The admin can specify the weight for each service. The ones with a higher weight
 4. Hash 
 This algorithm applies a hash function on the incoming requests’ IP or URL. The requests are routed to relevant instances based on the hash function result. 
 
-Dynamic Algorithms 
+5. Dynamic Algorithms 
```

Comment:
I believe this is another unordered list apart from `Static Algorithms`.

Reply:
Thank you.